cpp/dolfinx/io/ADIOS2Writers.h

```diff
@@ -147,6 +149,13 @@ class FidesWriter : public ADIOS2Writer
   /// @brief Write data with a given time
   /// @param[in] t The time step
   void write(double t);
+
+private:
+   /// @brief If true, the mesh is saved only at the first write and then reused,
```

Comment:
No need for doxygen docstrings on private data. Just use `//`.

Reply:
Done